docs/core/building/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Build .NET Core from source
+description: Learn how to build .NET Core and the .NET Core CLI from the source code
+keywords: .NET, .NET Core, source, build
+author: bleroy
+ms.author: mairaw
+ms.date: 03/29/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: 8b49079c-6ede-429a-92d7-ecd2fda1ab0e
+---
+
+# Build .NET Core from source
+
+The ability to build .NET Core from its source code is important in multiple ways: it makes it easier to port .NET Core to new platforms, it enables contributions and fixes to the product, and it enables the creation of custom versions of .NET.
+This article gives guidance to developers who want to build and distribute their own versions of .NET Core.
+
+## Build the CLR from source
+
+The source code for the .NET Core CLR can be found in [the `dotnet/coreclr` repository on GitHub](https://github.com/dotnet/coreclr/).
+
+The build currently depends on [Git](https://git-scm.com/), [CMake](https://cmake.org/), [Python](https://www.python.org/) and a C++ compiler.
```

Comment:
Perhaps turns this into a bullet list and explicitly call out these are the prerequisites